lib/db/mysql/patches/patch-005-006.sql

```diff
@@ -14,7 +14,7 @@ CREATE TABLE refreshTokens (
 
 -- Add expiresAt column for access tokens
 
-ALTER TABLE tokens ADD COLUMN expiresAt TIMESTAMP NOT NULL;
+ALTER TABLE tokens ADD COLUMN expiresAt TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP;
```

Comment:
That's because your mysql is a higher version. I believe we run 5.5 in production, and 5.6+ made this breaking change (yay).